runbot_gitlab/runbot_repo.py

```diff
@@ -279,25 +294,28 @@ def update(self):
         self._cr.commit()
         self._cr.autocommit(True)
 
-        # Put all protected branches as sticky
-        protected_branches = set(
-            b.name for b in project.find_branch(find_all=True, protected=True)
-        )
-        protected_branches.add(project.default_branch)
+        if self.sticky_protected:
+            # Put all protected branches as sticky
+            protected_branches = set(
+                b.name for b in project.find_branch(find_all=True, protected=True)
```

Comment:
While tests are broken for other reasons as well, flake8 does signal that this line is too long.